spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -143,17 +143,26 @@ public void testDryRun() throws IOException, InterruptedException {
         .execute();
     Assert.assertTrue("Default olderThan interval should be safe", Iterables.isEmpty(result1.orphanFileLocations()));
 
-    DeleteOrphanFiles.Result result2 = actions.deleteOrphanFiles(table)
-        .olderThan(System.currentTimeMillis())
-        .deleteWith(s -> { })
-        .execute();
+     DeleteOrphanFiles.Result result2 = actions.deleteOrphanFiles(table)
+         .olderThan(System.currentTimeMillis())
+         .deleteWith(s -> { })
+         .execute();
```

Comment:
Nit: Please make sure that any code you didn't change isn't updated. This makes it easier for people who maintain forks to cherry-pick from the open source into their fork.